runtime-modules/versioned-store-permissions/Cargo.toml

```diff
@@ -5,6 +5,7 @@ authors = ['Joystream contributors']
 edition = '2018'
 
 [dependencies]
+serde = { version = "1.0.101", features = ["derive"] }
```

Comment:
Add this line `'serde',` to the `std` feature.

Reply:
done in a498fef04691764c5dd9b9908785503586224590